src/server/game/Scripting/ScriptReloadMgr.cpp

```diff
@@ -868,8 +859,6 @@ class HotSwapScriptReloadMgr final
 
         Optional<fs::path> cache_path;
```

Comment:
Ok, but i don't understand where i need move the initialization (after bracket?) because initialization is also called in `auto module = ScriptModule::CreateFromPath(path, cache_path);`